spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestStructuredStreamingRead3.java

```diff
@@ -481,6 +481,35 @@ public void testReadStreamWithSnapshotTypeDeleteAndSkipDeleteOption() throws Exc
         .containsExactlyInAnyOrderElementsOf(Iterables.concat(dataAcrossSnapshots));
   }
 
+  @Test
+  public void testReadStreamWithSnapshotTypeDeleteAndSkipOverwriteOption() throws Exception {
+    table.updateSpec()
+        .removeField("id_bucket")
+        .addField(ref("id"))
+        .commit();
+
+    table.refresh();
```

Comment:
No need for refresh calls after altering a table. The table instance that was just committed will be up to date afterwards.

Reply:
I have removed the refresh call.